src/Program.spec.ts

```diff
@@ -702,6 +702,60 @@ describe('Program', () => {
             }]);
         });
 
+        it('accepts libpkg references script reference', () => {
+            program.setFile('components/component1.xml', trim`
+                <?xml version="1.0" encoding="utf-8" ?>
+                <component name="HeroScene" extends="Scene">
+                    <script type="text/brightscript" uri="libpkg:/components/component1.brs" />
+                </component>
+            `);
+            program.setFile('components/component1.brs', '');
+
+            program.validate();
+            expect(program.getDiagnostics()).to.be.empty;
+        });
+
+        it('does not set function not found diagnostic for function in libpkg referenced script reference', () => {
```

Comment:
Can you add some tests that verify these types of imports also work in .bs files.in import statements.

Reply:
Btw, I meant:

```
import "libpkg:/something.bs"
```